lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -598,6 +598,12 @@ def safely_acquire_lock_for_table(table, mode: :access_exclusive, &block)
     Thread.current[__method__] = nil unless nested_target_table
   end
 
+  # since rails version 6.0+ & 7.0 has bug which does not handle symbol for
+  # constraint name, we are converting name to string explicitly to solve that.
```

Comment:
Instead of specifying exactly 6.0 & 7.0, should it say something like `< 7.1`? I assume this was an issue in all versions prior to the fix in 7.1

Reply:
done https://github.com/braintree/pg_ha_migrations/pull/102/commits/33b712bc8cda3f8f91c4334f72a8b6c5a4849307#diff-e246218fba61b707b474c288f842dc0bf0c83436253bf2b49d05216d222f034cR601